mockwebserver/src/main/kotlin/mockwebserver3/QueueDispatcher.kt

```diff
@@ -59,6 +59,10 @@ open class QueueDispatcher : Dispatcher() {
     responseQueue.add(response)
   }
 
+  open fun clear() {
```

Comment:
Can you please implement this feature to maven lib please !

Reply:
do we have an eta on when this will be publicly available?